constellations/gemini-admin/src/ask_for_help/afh.sass

```diff
@@ -0,0 +1,54 @@
+@import "../_.constants.sass"
+
+#view-afh
+    width: 100%
+    padding: 0 $spacing2x
+    box-sizing: border-box
+
+    & h1
+        display: inline-block
+        margin: $spacing4x 0
+        line-height: $lineHeight3
+        font-size: $fontSize3
+
+    & ul
+        list-style: none
+        margin: 0
+        padding: 0
+    
+    & #header 
+        border-bottom: 2px solid $colorGrayDark
+    
+        & li
+            display: inline-block
+            height: $spacing4x
+            line-height: $spacing4x
+    
+    & li
+        margin: 0px $spacing1x
+    
+    .li-small
+        width: $spacing10x
+    .li-med
+        width: $spacing20x
+    .li-large
+        flex-grow: 1
+        width: $spacing48x
+    
+    & button
+        margin-top: $spacing2x
```

Comment:
Make this button follow the same style as the other pages. Make it a green background color with white font